timeshift-autosnap-apt

```diff
@@ -51,7 +51,17 @@ fi
 
 readonly SNAPSHOT_DESCRIPTION=$(get_property "snapshotDescription" "string" "{timeshift-autosnap-apt} {created before upgrade}")
 
-timeshift --create --comments "$SNAPSHOT_DESCRIPTION" || { echo "Unable to run timeshift-autosnap-apt! Please close Timeshift and try again. Script will now exit..." >&2; exit 1; }
+# This is a makeshift do-while loop, as per https://stackoverflow.com/a/16489942
+while true; do
+    SNAPSHOT_SUCCESSFUL=1
```

Comment:
This variable acts as exit condition of the do-while loop and tracks if a snapshot was taken successfully. For now, it is 1 ("false") since we didn't take any snapshot.